weos_api.go

```diff
@@ -282,7 +282,7 @@ func (p *API) ResponseRecording(next echo.HandlerFunc) echo.HandlerFunc {
 }
 
 func (p *API) HealthChecker(c echo.Context) error {
-	return c.String(http.StatusOK, "Hello, World!")
+	return c.String(http.StatusOK, "Hello, World! "+"Version: "+p.Config.Version)
```

Comment:
Switch to using a json response so that way something else could parse the response.  Get rid of the hello world
```
{
    "version": "1.0.2"
}
```